.github/workflows/numpy2_compatibility.yml

```diff
@@ -5,10 +5,9 @@ name: Numpy2 compatibility
 
 on:
 
-    # Comment before merging in main
-    # pull_request:
-    #    branches:
-    #    -   '*'
+    pull_request:
```

Comment:
oops, this seems to be unrelated.

Reply:
Actually this is intentional: this is the 3rd time that I have had to go and manually check in the action tab to see if something numpy2 related was broken. I'd rather have this check run regularly for better automatic early detection.

By doing this it will be checked with every PR: this would also ensure that every PR does not bring changes that would be numpy2 incompatible.